nd4j/nd4j-backends/nd4j-backend-impls/nd4j-native/src/main/java/org/nd4j/linalg/cpu/nativecpu/blas/SparseCpuLevel2.java

```diff
@@ -31,7 +31,6 @@
  * @author Audrey Loeffel
  */
 public class SparseCpuLevel2 extends SparseBaseLevel2 {
```

Comment:
do we actually want to keep this class? in medium-to-long term all this stuff should be moved to C++ anyway.

Reply:
No, this is also to be deleted as refactoring continues.